config/workload/auctionmark.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration xsi:noNamespaceSchemaLocation="workload.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <workloads>
+        <workload benchmarkClass="com.oltpbenchmark.benchmarks.auctionmark.AuctionMarkBenchmark">
+            <scaleFactor>1</scaleFactor>
+            <terminals>1</terminals>
+
+            <phases>
+                <phase>
+                    <time>60</time>
+                    <rateType>LIMITED</rateType>
```

Comment:
This seems to be a new parameter; trying to find the documentation through github's file browser (if any), I assume the only other option would be called something like `UNLIMITED` in which case `<rate>` is ignored?

Reply:
great catch... this was one of the harder sections of DBWorkload to port over.  There was old logic like this...

```
 } else if (rate_string.equals(RATE_UNLIMITED)) {
                    rateLimited = false;
}...
```

my thinking here was that instead of inferring the ratetype by the presence or absence of UNLIMITED, we ask users to be explicit, therefore this element is always required and the logic becomes clearer.  In the new world a phase can be LIMITED, UNLIMITED or DISABLED.  hopefully this explanation helps and the logic is sound.  Have a look at the new method ` private static List<Phase> getPhases(Workload workload)` on DBWorkload.